Sites/GD2/Project 1/Styles.css

```
@@ -0,0 +1,105 @@
+img {
+    max-width: 100%
+}
+
+h1 {
+    color: #260648;
+    font-weight: bold;
+    font-size: 40px
+}
+
+bodycolor {
+    color: #F6F6EC
+}
+
+h2 {
+    color: #6B4B8C;
+    font-weight: bold;
+    font-size: 20px;
+    font-alignment: left
+}
+
+div {
+    margin: 12px
+}
+
+width=39em p {
```

Looks like this bit `width=39em` escaped from somewhere else in your file?

```suggestion
p {
```